service/policies/entitlements/entitlements-keycloak.rego

```diff
@@ -24,10 +16,18 @@ else := {"entities": [{
 	"userName": input.entity.username,
 }]} if { input.entity.username }
 
+
 attributes := [attribute |
 	# external entity
-    response := keycloak.resolve.entities(idp_request, idp_config)
-    entity_representations := response.entityRepresentations
+    response := http.send({
+        "method" : "POST",
+        "url": input.ers_url,
+        "body": ers_request
+    })
```

Comment:
@elizabethhealy 
How is an auth header being added to this request? Or are we assuming this will be a public endpoint?

Reply:
ooh this is a good question, i usually have auth disabled and forget about it.
we could assign a client to the authorization service and have that client retrieve a token under getEntitlements thats then passed into opa
other option is guess is to pass the authorization serivice assigned client and secret into opa and retrieve it from there but seems less safe

Reply:
@strantalis is calling the token endpoint in authroization service then passing the token to rego any better? that way i guess we wouldnt have to pass the client credentials to rego to get the token